scheduled_lambdas.go

```diff
@@ -0,0 +1,166 @@
+package rockset
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/rs/zerolog"
+
+	rockerr "github.com/rockset/rockset-go-client/errors"
+	"github.com/rockset/rockset-go-client/openapi"
+	"github.com/rockset/rockset-go-client/option"
+)
+
+// CreateScheduledLambda creates a new scheduled lambda, with an optional description.
```

Comment:
you can set a description
```suggestion
// CreateScheduledLambda creates a new scheduled lambda, with optional arguments.
```